schemas/v1.0/schema.yaml

```
@@ -0,0 +1,563 @@
+$id: 'https://spec.openapis.org/arazzo/1.0/schema/2024-08-01'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+description: |-
+  The description of OpenAPI Initiative Arazzo v1.0.0 documents
+  without schema validation, as defined by https://spec.openapis.org/arazzo/v1.0.0
+type: object
+properties:
+  arazzo:
+    description: The version number of the Arazzo Specification
+    type: string
+    pattern: '^1\.\d+\.\d+(-.+)?$'
```

```suggestion
    pattern: '^1\\.0\\.\\d+(-.+)?$'
```
suggest limiting to 1.0.x